### PR TITLE
Add new patterns to detect the YII framework

### PR DESCRIPTION
**Pull request by craiglondon**

### CODE REVIEW

**Comment by craiglondon** [comment]
Testable here   https://www.peopleperhour.com/  for pattern  `<input type=\"hidden\" value=\"[a-zA-Z0-9]*\" name=\"YII_CSRF_TOKEN\" \/>`

File: src/apps.json [hunk 1]
```diff
@@ -8717,7 +8717,10 @@
 				18
 			],
 			"html": [
-				"Powered by <a href=\"http://www.yiiframework.com/\" rel=\"external\">Yii Framework</a>"
+				"Powered by <a href=\"http://www.yiiframework.com/\" rel=\"external\">Yii Framework</a>",
+				"<input type=\"hidden\" value=\"[a-zA-Z0-9]*\" name=\"YII_CSRF_TOKEN\" \/>",
```
Comment (jvoisin) [inline src/apps.json]: It seems that the size of the anti-csrf token is fixed, so `"<input type=\"hidden\" value=\"[a-zA-Z0-9]{4}\" name=\"YII_CSRF_TOKEN\" \/>"` will be a bit better :)

File: src/apps.json [hunk 2]
```diff
@@ -8717,7 +8717,10 @@
...
+				"<!\\[CDATA\\[YII-BLOCK-HEAD\\]\\]>",
+				"<!\\[CDATA\\[YII-BLOCK-BODY-BEGIN\\]\\]>"
```
Comment (jvoisin) [inline src/apps.json]: What about `"<!\\[CDATA\\[YII-BLOCK-(?:HEAD|BODY-BEGIN)\\]"` instead?
It will go a bit faster ;)

Comment (craiglondon) [inline src/apps.json]: @jvoisin What Regex tester do you usually use?  I usually use http://regexr.com/, but then usually I have to escape some stuff to make it compatible with JSON.

File: src/apps.json (hunk 1 above)
Comment (craiglondon) [inline src/apps.json]: @jvoisin The pattern proposed above doesn't match
`<input type="hidden" value="82f65c11d32bf867095aa35b683de4b8a5b3f49f" name="YII_CSRF_TOKEN" />`, unless I'm doing something wrong.  Please advise.

Comment (jvoisin) [inline src/apps.json]: It should be `"<input type=\"hidden\" value=\"[a-zA-Z0-9]{40}\" name=\"YII_CSRF_TOKEN\" \/>"`, I forgot a zero, sorry :)

Comment (craiglondon) [inline src/apps.json]: Thank you.  I updated the pattern.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**